applications/sasquatch/values-base.yaml

```diff
@@ -259,6 +259,15 @@ telegraf:
       topicRegexps: |
         [ "lsst.obsenv" ]
       debug: true
+    scimma:
+      enabled: true
+      database: "lsst.scimma"
+      timestamp_format: "unix_ms"
+      timestamp_field: "timestamp"
+      topicRegexps: |
+        [ "lsst.scimma" ]
+      debug: true
+      offset: newest
```

Comment:
I would use "oldest" unless you have a strong reason for using "newest" as explained above.